pyvista/plotting/background_renderer.py

```diff
@@ -70,7 +70,7 @@ def resize(self, *args):
             self._prior_window_size = self.parent.window_size
 
         actor = self._actors['background']
-        image_data = actor.GetInput()
+        image_data = actor.GetInput()  # type: ignore[attr-defined]
```

Comment:
I tried to find `self._actors['background']` elsewhere to get the type from wherever this is set. But, I couldn't find it being set anywhere else in the code. Not sure this has test coverage either.... might be worth investigating as a follow up

Reply:
Was stumped for a second, but turns out it's added in this class (above) via:
```py
self.add_actor(image_actor, name='background')
```
Where `name=` is used as the key in the dict. However, you're right that it's not triggered by any of our tests.

Reply:
Good find, I guess this code should probably first check that `'background'` is key and that it's also a `vtkImageActor` instance (the `# type: ignore[attr-defined]` that I added indicates that `GetInput` is not a `vtkActor` method)

Should we leave the `type: ignore` and address this separately?

Reply:
Yes, it will make the blame easier to navigate.